app/react/Metadata/helpers/formater.js

```diff
@@ -57,6 +70,10 @@ const conformSortedProperty = (metadata, templates, doc, sortedProperty) => {
     result = addCreationDate(result, doc);
   }
 
+  if (sortedProperty === 'editDate') {
+    result = addModificationDate(result, doc);
+  }
```

Comment:
@grafitto not tested.

Reply:
I believe I also have added a test to this.